CONFIG.md

```
@@ -45,13 +45,23 @@ its own full time window.
 
 | Option | Type | Default | Description |
 |--------|------|---------|-------------|
-| `--streaming-output` | String | - | File for streaming results during execution |
+| `--streaming-output-json` | String | - | JSON file for streaming per-message results |
+| `--streaming-output-csv` | String | - | CSV file for streaming per-message results |
 | `--continue-on-error` | Boolean | `false` | Continue running if one test fails |
-| `--verbose` | `-v` | Boolean | `false` | Enable verbose output |
+| `--verbose` | `-v` | Flag | *(repeatable)* | Increase log verbosity (-v=DEBUG, -vv=TRACE) |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 4; Actual: 5; Too many cells, extra data will be missing

(MD056, table-column-count)